components/core/input/primitives.ts

```diff
@@ -132,3 +132,17 @@ export const Tissue = {
     },
   },
 } as Primative
+
+export const TumorGeneExpression = {
+  name: 'TumorGeneExpression ',
+  label: 'TumorGeneExpression ',
+  icon: [tissue_icon],
+  color: '#98D7C2',
+  extra: {
+    term: {
+      meta: {
+        example: 'Brain',
```

Comment:
This should probably be a tumor name as an example, perhaps `Meningioma`, for instance.

Reply:
Corrected copy pasta error with https://github.com/nih-cfde/playbook-partnership/pull/38/commits/06be34924211e23c6a4429ffb00282e73bcf78ae.